model/postgres/postgres_timeline.rb

```diff
@@ -20,6 +21,30 @@ def bucket_name
     ubid
   end
 
+  def generate_walg_config
+    <<-WALG_CONF
+WALG_S3_PREFIX=s3://#{ubid}
+AWS_ENDPOINT=#{Config.postgres_service_blob_storage_endpoint}
+AWS_ACCESS_KEY_ID=#{access_key}
+AWS_SECRET_ACCESS_KEY=#{secret_key}
+AWS_REGION: us-east-1
```

Comment:
huh, does wal-g not have some way to be more lucid with minio than pretending w.r.t. region? I know they use it for testing.

Reply:
Unfortunately no. This is even in their docs; https://wal-g.readthedocs.io/STORAGES/#examples